### DISCUSSION TITLE
[Feature] Suggestions on distinguishing whether services have been disabled in UI

### DISCUSSION THREAD

**Discussion by ZY945** [apache/skywalking#12765] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

I found that the data still exists in the UI after the service is stopped. The related issue identifier can be set to TTl.
[https://github.com/apache/skywalking/discussions/9390](https://github.com/apache/skywalking/discussions/9390)
Is there currently an idea to differentiate whether services have been discontinued in the UI?

### Are you willing to submit a pull request to implement this on your own?

- [X] Yes I am willing to submit a pull request on my own!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
No, we are not doing that. You could see metrics on your own.
And this is not an issue.
